src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java

```diff
@@ -151,6 +149,10 @@ else if (structureBlock instanceof BlockStairs && structureBlockState.equals(wor
             {
                 return true;
             }
+            else if((structureBlock == Blocks.DIRT || structureBlock == Blocks.GRASS) && (worldBlock == Blocks.DIRT || worldBlock == Blocks.GRASS))
+            {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This branch's code block is the same as the block for the branch on line 149. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1871)